src/main/resources/sample/failing/TECHNICAL/doesnt-pass-regex-validation.json

```diff
@@ -0,0 +1,12 @@
+{
+  "requiredStringField": "requiredStringValue",
+  "requiredArray": [
+    "requiredArrayValue1",
+    "requiredArrayValue2",
+    "requiredArrayValue3"
+  ],
+  "phoneNumberField": "invalid_value",
+  "dateField": "2020 jan 1",
+  "emailField": "email address@domain",
```

Comment:
rajouter un test sur "xxx email@quipassela.regex xxx" pour s'assurer qu'on gère bien les $ ^ 
après => peut-être le mettre dans valid pour le moment avec un todo "should be failing" et comme ça on le migrera ? Ou le mettre dans la tache ?

Reply:
Un ticket asana existe deja: https://app.asana.com/0/1203993034002721/1207853269605351/f , je pense que c'est pas la peine d'ajouter un todo